includes/Feature.php

```diff
@@ -131,7 +131,15 @@ final public function isEnabled() {
 	 * @return bool True if the feature toggle is allowed, false otherwise.
 	 */
 	public function canToggleFeature() {
-		return (bool) current_user_can( 'manage_options' );
+		return (bool) apply_filters(
+			// specific feature canToggle filter
+			"newfold/features/filter/canToggle:{$this->name}",
+			apply_filters(
+				// generic canToggle filter
+				'newfold/features/filter/canToggle',
+				current_user_can( 'manage_options' )
```

Comment:
@circlecube Given that the idea was that someone would override the `canToggleFeature` method, perhaps we should create a separate, private method that handles the filters and just have this method handle the permission logic.